src/main/java/com/codecool/virtualpub/data/Bartender.java

```diff
@@ -1,18 +1,30 @@
 package com.codecool.virtualpub.data;
 
+import java.util.Random;
+
 public class Bartender {
     private String name;
+    private final Random random;
 
     public Bartender(String playerName) {
         this.name = playerName;
+        this.random = new Random();
     }
-
-
-    public String getName() {
-        return name;
+    
+    public DrinkType getDrinkType() {
+        // todo get from player in console
+        return DrinkType.values()[random.nextInt(DrinkType.values().length)];
     }
 
-    public void pourDrink(){
+    public int getDrinkAmount() {
+        // todo get from player in console
+        return 1;
+    }
 
+    public void welcome() {
+        int idx = random.nextInt(4);
+        String welcome = BartenderScript.WELCOME.getScripts().get(idx);
```

Comment:
I don't really get this line. Why are we getting this line? In the next line you are giving the 'welcome' variable a new value anyway.